source/docs/articles/users/access-management.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Access Management for Site Owners and Administrators
+description: Learn how to remove a team member or outside resource from a site.
+keywords: remove team member, delete team member, delete user
+---
+
+When a person with access to your site(s) on the platform leaves the company or project, it is important to immediately remove them from the team so that they no longer have access to make changes to your site.
+
+After a user leaves, we recommend you:
+
+- Delete or block the user's account in Drupal or WordPress.
```

Comment:
Add a sub-section linking to drupal/wordpress docs for user management or removal.